Sources/swift-bundler/Bundler/SwiftPackageManager/SwiftPackageManagerError.swift

```diff
@@ -57,6 +58,11 @@ enum SwiftPackageManagerError: LocalizedError {
         // make that weak assumption about the implementation
         return
           "Failed to parse package manifest output: \(error?.localizedDescription ?? "Unknown error")"
+      case .failedToParsePackageManifestToolsVersion(let fallbackVersion, let error):
+        return """
+          Failed to parse package manifest tools version: \(error?.localizedDescription ?? "Unknown error")
+          Falling back to tools version \(fallbackVersion), which may fail to build with your target.
```

Comment:
The code that handles the tools version parsing throws this error instead of using the fallback. I think we should assume that the package won't build anyway if the tools version can't be passed, so throwing an error is correct. The error just shouldn't say that it's falling back given that it's actually given up.

Btw, thanks for implementing tools version parsing! That's a good improvement over the old hardcoding approach lol.

Reply:
Oh right lol, cleaned it up! Thanks